src/rtprecv.c

```diff
@@ -451,11 +473,11 @@ void rtprecv_decode(const struct sa *src, const struct rtp_header *hdr,
 			return;
 	}
 
+	if (flush)
+		rtprecv_resync(rx, hdr);
```

Comment:
I think now it should be okay if we call `rtprecv_resync()` directly in line 466 and drop the `bool flush`. Note, that the payload handler needs `pt` and `mb` not more. And `mb` is not flushed. So it should be safe. Anyway, the payload handler in audio.c runs in another thread.
Be careful with the lock!